Sources/Benchmark/OperatingSystemStats/OperatingSystemStatsProducer+Darwin.swift

```diff
@@ -141,6 +148,30 @@ final class OperatingSystemStatsProducer {
                                          writeBytesPhysical: 0)
 
         return stats
+#else
+        lock.lock()
+        let threads = peakThreads
```

Comment:
These will always be zero as they are measured above by the sampler thread and using libproc information about the threads - so can as well drop this lock and just return OperatingSystemStats that is fully 0.

Also change` metricSupported` to always return false on `.iOS`

Reply:
Updated.